app/components/TaskDetailsSidebar/style.scss

```diff
@@ -71,6 +71,7 @@ $sidebar-components-vertical-padding: 25px;
 
   &__sub-tasks {
     padding-bottom: $sidebar-components-vertical-padding;
+    .swimlane__header__button {display: none;}
```

Comment:
Each css selector should have a new line after the `{` and before the `}` like so:
```
.swimlane__header__button {
  display: none;
}
```